airflow/providers/databricks/operators/databricks.py

```diff
@@ -316,6 +316,10 @@ def execute(self, context: Context) -> int:
         if job_id is None:
             return self._hook.create_job(self.json)
         self._hook.reset_job(str(job_id), self.json)
+        if "access_control_list" in self.json and self.json["access_control_list"] is not None:
```

Comment:
```suggestion
        if self.json.get("access_control_list") is not None:
```

Reply:
Yep, i think this is better 👍  but I noticed this practice is used elsewhere. Should we change all of them into this style? (maybe create an issue?)

Reply:
Python 3.8+ there 😄 

```python
if (access_control_list := self.json.get("access_control_list")) is not None:
    acl_json = {"access_control_list": access_control_list}
    self._hook.update_job_permission(normalise_json_content(acl_json))
```

Reply:
It's indeed even better 👍